mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```diff
@@ -170,13 +177,37 @@ GemmRewritePattern::matchAndRewrite(GemmOp op, GemmOpAdaptor adaptor,
   // Note, matrix dimension correctness is handled in the verifier
   GemmSize size(/*g=*/aShape[0], /*m=*/aShape[2], /*k=*/aShape[1],
                 /*n=*/bShape[2]);
+
+  GemmSize scale{1, 1, 1, 1};
```

Comment:
Im just curious to know why are we are adding a generic scaling for all dims ?
(Id rather not add something like that unless we have some long term plan)